src/com/projectkorra/projectkorra/earthbending/metal/MetalClips.java

```diff
@@ -228,7 +229,7 @@ public void launch() {
 		dz = target.getZ() - location.getZ();
 		final Vector vector = new Vector(dx, dy, dz);
 		vector.normalize();
-		this.targetEntity.setVelocity(vector.multiply(this.metalClipsCount / 2D));
+		GeneralMethods.setVelocity(this,	this.targetEntity, vector.multiply(this.metalClipsCount / 2D));

```

Comment:
Please change to `GeneralMethods.setVelocity(this, this.targetEntity, vector.multiply(this.metalClipsCount / 2D));`